sdk/daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -287,6 +287,10 @@ class Engine(val config: EngineConfig = Engine.StableConfig, allowLF2: Boolean =
           )
     } yield validationResult
 
+  def translateKey(identifier: Identifier, key: Value): Result[Value] = {
```

Comment:
Why return a plain `Value` instead of a `GlobalKey` here?

Reply:
Just 2.x change minimization. For 3.x I would return a `GlobalKey`.